string id_pareto_front
int32[] num_individuals
string[] id_pareto_inds
string storage_position

float64[] reference_point
int32 num_ind_ref_point
string id_pareto_front_ref

float64[] close_individual
int32 num_ind_close
string id_pareto_front_close


